executor/aggregate_test.go

```diff
@@ -384,6 +386,9 @@ func (s *testSuite1) TestGroupConcatAggr(c *C) {
 	// issue #5411
 	tk := testkit.NewTestKit(c, s.store)
 	tk.MustExec("use test")
+	// https://github.com/pingcap/tidb/issues/6838 isn't implemented, so use this trick.
+	tk.Se.GetSessionVars().InitChunkSize = 3
+	tk.Se.GetSessionVars().MaxChunkSize = 3
```

Comment:
why set it to `3`?

Reply:
using default value `2` will be broken after https://github.com/pingcap/tidb/pull/8903 shipped.

hashaggExecutor request 2 record per time from indexlookupExecutor, so indexlookupExecutor will return 2 rounds as [2 records, 1 record], hashagg will concurrent running two result, at last have chance be wrong order(1 record come first). use 3 to force indexlookupExecutor return 1 round and no order question.

... group_concat doesn't impl `order by`, so we only can use this to force order in testcase